tools/aws_lab_setup/roles/common/tasks/RedHat.yml

```diff
@@ -11,11 +11,13 @@
     state: present
   tags:
     - common
+  when: localsecurity
```

Comment:
This seems like one knob we could have skipped to me. Irregardless, please use a block to eliminate these redundant conditionals.